### ISSUE TITLE
Incompatibility with Sodium@1.16.x/next

### ISSUE THREAD

**Issue by magneticflux-** [Minenash/Seamless-Loading-Screen#25]
After commit https://github.com/CaffeineMC/sodium-fabric/commit/19ee2f0b70ae7c5f3c5f76c06e8bcf3adb8d69b8, the transparent loading screen mixin can't be applied.

As a workaround, you can add a custom property in your `fabric.mod.json` as shown here: https://github.com/CaffeineMC/sodium-fabric/blob/9915466a81e9d5e32f03fe16b1902c1e4880d824/src/main/java/me/jellysquid/mods/sodium/common/config/SodiumConfig.java#L97-L114

The property would look like this:
```json
"custom": {
    "sodium:options": {
        "mixin.features.gui.fast_loading_screen": false
    }
}
```

**Comment by jowsey** [comment]
Thank you for the workaround!

**Comment by Minenash** [comment]
Thanks for the report and a fix! I'll push an update tonight with it

**Comment by Minenash** [comment]
Fixed in v1.3.2
